### PR TITLE
Kazakh Cyrillic and Latin layouts

### PR DESCRIPTION
**Pull request by tenextractor**
* Cyrillic
image

* Latin
image

### CODE REVIEW

File: srcs/layouts/latn_qwerty_kk.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <key shift="0.5" key0="ä"/>
+    <key key0="ñ"/>
+    <key key0="ğ"/>
+    <key key0="ç"/>
+    <key key0="ū"/>
+    <key key0="ü"/>
+    <key key0="ı"/>
+    <key key0="ö"/>
+    <key key0="ş"/>
+  </row>
```
Comment (Julow) [inline srcs/layouts/latn_qwerty_kk.xml:17]: These letters are present on the layout at two position at the same time: `ä, ç, ñ, ö, ü, ğ, ı, ş, ū`. It seems that the top row could be entirely removed.

File: srcs/layouts/cyrl_jcuken_kk.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <key key0="ә"/>
+    <key key0="і"/>
+    <key key0="ң"/>
+    <key key0="ғ"/>
+    <key key0="ү"/>
+    <key key0="ұ"/>
+    <key key0="қ"/>
+    <key key0="ө"/>
+    <key key0="һ"/>
+    <key key0="ъ"/>
```
Comment (Julow) [inline srcs/layouts/cyrl_jcuken_kk.xml:14]: `ъ` is present twice on the layout.

File: srcs/layouts/cyrl_jcuken_kk.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="ЙЦУКЕН (Қазақша)" script="cyrillic">
+  <row>
+    <key key0="ё"/>
```
Comment (Julow) [inline srcs/layouts/cyrl_jcuken_kk.xml:4]: `ё` is present twice on the layout.

File: srcs/layouts/latn_qwerty_kk.xml (hunk 1 above)
Comment (tenextractor) [reply srcs/layouts/latn_qwerty_kk.xml]: Removed them from swipe keys

File: srcs/layouts/cyrl_jcuken_kk.xml (hunk 2 above)
Comment (tenextractor) [reply srcs/layouts/cyrl_jcuken_kk.xml:14]: Removed both Ё and Ъ

**Comment by Julow** [comment]
Alright, let's merge!

**Pull request merged**